### DISCUSSION TITLE
TMin and TMincool

### DISCUSSION THREAD

**Discussion by eshroom** [revk/xESP32-Faikout#370] [Q&A]
I am trying to achieve a minimum allowed cooling temp. I was not sure which of the two would achieve these, so I set both to 19, yet through the Madoka controller we are still able to set a lower temperature e.g. 17 and it shows up in HA and the faikin control page correctly as 17.

Do Tmin and Tmincool not control the minimum temp? Thanks.

**Comment by revk** [comment]
I'd have to check the code, but they will only control what it set through the Faikin.
